package.json

```diff
@@ -32,6 +32,10 @@
     "scripts": {
         "prChecks": "pnpm install && pnpm lint && pnpm checkFormat && pnpm testRepo && pnpm buildRepo",
         "build": "ts-node ./dev/scripts/build.ts",
+        "clean": "pnpm run \"/^clean:.*/\" && pnpm --recursive --parallel --stream run clean",
+        "clean:build": "rm -f tsconfig.temp.json && rm -f tsconfig.temp.tsbuildinfo && rm -rf dist",
+        "clean:deps": "rm -rf node_modules",
+        "reboot": "pnpm clean && pnpm install && pnpm build",
```

Comment:
Something like this could be useful, but since a clean should be built into build anyways, I'm leaning toward removing it for now. I've had a lot of scripts over time I thought I would sometimes use but in general if I don't have frequent cause to use them, I just end up forgetting about them and they end up cluttering the `package.json`.

Plus we have to worry about Windows compatibility etc. Sorry 😔